components/new-member-list-item/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from './new-member-list-item.module.scss';
+import { motion } from 'framer-motion';
+
+const NewMemberListItem = ({ newMemberDetails }) => {
+  const { username } = newMemberDetails;
+  const fullName = `${newMemberDetails.first_name + ' ' + newMemberDetails.last_name}`;
```

Comment:
NIT: why not destructure first_name and last_name as well at line no 7?

Reply:
will be done